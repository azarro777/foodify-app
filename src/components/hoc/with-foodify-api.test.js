import React from 'react';
import WithFoodifyApi from './with-foodify-api';
import { shallow } from 'enzyme';

describe('Testing <WithFoodifyApi/> component', () => {
  it('WithFoodifyApi have rendered correctly', () => {
    const foodify = shallow(<WithFoodifyApi />);
    expect(foodify).toMatchSnapshot();
  });
});
