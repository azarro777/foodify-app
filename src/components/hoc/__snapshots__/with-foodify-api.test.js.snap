// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <WithFoodifyApi/> component WithFoodifyApi have rendered correctly 1`] = `ShallowWrapper {}`;
